docs/api/app.md

```diff
@@ -153,9 +153,23 @@ Returns:
 
 * `event` Event
 
-Emitted when mac application become active. Difference from `activate` event is
-that `did-become-active` is emitted every time the app becomes active, not only
-when Dock icon is clicked or application is re-launched.
+Emitted when the application becomes active (i.e., the macOS title bar is
+showing the name of the app). The difference between this and the `activate`
+event is that `did-become-active` is emitted every time the app becomes active,
+whereas `activate` is only emitted when the Dock icon is clicked or the
+application is re-launched.
+
+See [`-[NSApplicationDelegate applicationDidBecomeActive:]`](https://developer.apple.com/documentation/appkit/nsapplicationdelegate/1428577-applicationdidbecomeactive?language=objc)
+
+### Event: 'did-resign-active' _macOS_
+
+Returns:
+
+* `event` Event
+
+Emitted when the application loses active state.
```

Comment:
small nit: i think we should include both that the app is no longer active and doesn’t have focus (as apple docs do)